book/data_sources.qmd

```diff
@@ -1,37 +1,32 @@
+# Data sources
 
-# Data Sources
+This page tabulates and provides links to all data sources used in this project.
+For more details on how each dataset is used and processed, refer to the [Methodology](method.qmd) section.
 
-The list of data sources used in this project. For more details on how is each dataset
-used and processed, refer to the [Methodology](method.qmd).
-
-## List of data source
-
-Below is a list of data sources organised by the purpose
-
-### Project-wide
+## Project-wide
 
 - OS | 2011 Census Output Areas | [link](https://geoportal.statistics.gov.uk/datasets/ons::output-areas-dec-2011-boundaries-ew-bgc/about)
 
-### Accessibility indicators
+## Accessibility indicators
 
 - ONS | Population Weighted Centroids (PWC) of Output Areas | [link](https://www.ons.gov.uk/census/2001censusandearlier/dataandproducts/outputgeography/outputareas)
```

Comment:
The PWC link doesn't actually link to any PWCs, it links to an explanation of output areas.

I did find a PWC dataset for 2011 here: https://geoportal.statistics.gov.uk/datasets/ons::output-areas-dec-2011-pwc/about Would this (or something else) be a more suitable link?